app/budget/directives/budgetScenarioToolbar/budgetScenarioDropdown/budgetScenarioDropdown.css

```diff
@@ -32,3 +32,21 @@
 	display: flex;
 	justify-content: space-between;
 }
+
+@media print {
+	@page {
+		margin: 1cm;
+	}
+
+	.do-not-print {
+			display: none !important;
```

Comment:
Is the important flag absolutely necessary? And if so, why is that?

Using the important flag quickly complicates CSS and makes future work harder. It should be used as a last resort.

Reply:
No need of !important flag